components/cam/bld/namelist_files/namelist_defaults_cam.xml

```diff
@@ -1045,6 +1045,7 @@
 <micro_tend_output       	      > .true.  </micro_tend_output>
 <p3_qc_autoCon_Expon    		      >  2.47   </p3_qc_autoCon_Expon>
 <p3_qc_accret_Expon    		      >  1.15   </p3_qc_accret_Expon>
+<do_prescribed_CCN>                    > .true. </do_prescribed_CCN>
```

Comment:
we should probably make the default behavior false for now so it doesn't change under people who aren't explicitly expecting the change.

Reply:
Changed to false by default.